api/config/packages/sentry.yaml

```diff
@@ -3,8 +3,9 @@ sentry:
     register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
     options:
         environment: '%env(SENTRY_ENVIRONMENT)%'
-        integrations:
-            - 'Sentry\Integration\IgnoreErrorsIntegration'
```

Comment:
We used `IgnoreErrorsIntegration` before to filter out 4xx error (see further below in this file). We should add these exceptions again.

Reply:
done